docs/components/input.md

```diff
@@ -100,6 +100,29 @@ You also have the option of using the `.help` utility class. This is meant to be
 </template>
 ```
 
+### characterLimit
+
+Use this prop to specify a character limit for the input.
+
+<KInput model-value="This field has too many characters" :character-limit="10" class="w-100" placeholder="Placeholder text" />
+
+```vue
+<KInput model-value="This field has too many characters" :character-limit="10" class="w-100" placeholder="Placeholder text" />
+```
+
+If the `characterLimit` is exceeded, the character counter below the `KInput` will override the display of a provided `errorMessage` until the character count is within the acceptable range.
```

Comment:
May want to note that the character counter is only visible when the char limit is exceeded

Reply:
Added